internal/pkg/service/stream/storage/level/local/encoding/encoder/csv/csv_test.go

```diff
@@ -40,159 +32,27 @@ type fileCompression struct {
 	FileDecoder func(t *testing.T, r io.Reader) io.Reader
 }
 
-func TestCSVWriter_InvalidNumberOfValues(t *testing.T) {
-	t.Parallel()
```

Comment:
Test in bad place, moved here:
https://github.com/keboola/keboola-as-code/blob/a9357dd8d9d85bc599aae477948d0a3709b5d498/internal/pkg/service/stream/storage/level/local/encoding/pipeline_test.go#L148-L164